cors.go

```diff
@@ -92,12 +104,18 @@ func Default() *Cors {
 func (cors *Cors) Handler(h http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		if r.Method == "OPTIONS" {
+			if cors.options.Debug {
+				cors.options.log.Printf("Handler: Preflight request")
+			}
```

Comment:
Why not moving those traces in the called method instead of repeating them everywhere they are called?